middleware/middleware.go

```diff
@@ -46,3 +50,17 @@ func newTokenMiddleware(verify authentication.TokenAuthentication, extractors ..
 		})
 	}
 }
+
+// AuthFuncGRPC returns a new grpc_auth.AuthFunc to use with the gazebo-web authentication library.
```

Comment:
This documentation can include an example on how to use it with the grpc interceptor.